torch/utils/data/datapipes/iter/routeddecoder.py

```diff
@@ -10,7 +11,7 @@
 )
 
 
-@functional_datapipe('decode')
+@functional_datapipe('routed_decode')
```

Comment:
Is there anyone using `.decode` before? Will this break anything for users?

Reply:
I am removing it because no one from domains really uses it 😢